src/DropdownButton.tsx

```diff
@@ -2,6 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import Dropdown from './Dropdown';
+import DropdownToggle from './DropdownToggle';
+import { BsPrefixRefForwardingComponent } from './helpers';
+import DropdownMenu from './DropdownMenu';
+
+type PropsFromToggle = Partial<
+  Pick<
+    React.ComponentPropsWithRef<typeof DropdownToggle>,
+    'href' | 'size' | 'variant' | 'disabled'
```

Comment:
maybe we can just export this subset from DropdownToggle?

Reply:
Yeah, or rather `PropsFromToggle` altogether – this is used in multiple places, it looks like :)